packages/x-components/src/x-modules/related-prompts/components/related-prompts-list.vue

```diff
@@ -66,23 +69,48 @@
       showOnlyAfterOffset: {
         type: Boolean,
         default: false
-      }
+      },
+      customQuery: String
```

Comment:
I'd like to avoid custom prefix everything. I think we could just use query for the prop